cli/fossilize_replay.cpp

```diff
@@ -2467,6 +2447,60 @@ struct ThreadedReplayer : StateCreatorInterface
 		return !info.info || (info.info->pLibraryInfo && info.info->pLibraryInfo->libraryCount != 0);
 	}
 
+	template <typename DerivedInfo>
+	uint32_t pipeline_library_get_parent_depth(const VkPipelineLibraryCreateInfoKHR *library,
+	                                           const vector<DerivedInfo> *deferred,
+	                                           const unordered_map<Hash, DerivedInfo> &parents)
+	{
+		uint32_t max_depth = 0;
+
+		for (uint32_t i = 0; i < library->libraryCount; i++)
```

Comment:
I don't think there is even any guarantee here that a derived pipeline has been parsed, so I don't understand how you can compute the parent depth.